includes/upload_page.form.inc

```diff
@@ -28,7 +28,7 @@ function islandora_paged_content_upload_page_form(array $form, array &$form_stat
   $extensions = array('tiff tif jp2 jpg jpeg');
   $languages = module_exists('islandora_ocr') ? islandora_ocr_get_enabled_tesseract_languages() : array();
```

Comment:
Languages is being handled differently here than in the previous form. Lacks a default.

Reply:
Aha; adding this in now

Reply:
@qadan also: on line 22. There is a module load include and then on line 29 a check if islandora OCR does exist. Wonder if the module load include will fail if ocr does not exist? Any ideas?

Reply:
Yes, I believe that's an issue. Resolved it in the current PR